applications/crossbar/src/modules/cb_clicktocall.erl

```diff
@@ -447,6 +449,7 @@ build_originate_req(Contact, Context) ->
        ,{<<"Outbound-Callee-ID-Number">>, CalleeNumber}
        ,{<<"Outbound-Caller-ID-Name">>, FriendlyName}
        ,{<<"Outbound-Caller-ID-Number">>, OutboundNumber}
+	   ,{<<"Outbound-Call-ID">>, CallId}
```

Comment:
@fenollp @k-anderson looking at https://github.com/2600hz/kazoo/pull/1233 it seems that it sets the outbound-call-id per endpoint but with the same call-id for all endpoints